text_docs/intro_article.md

```diff
@@ -1,3 +1,6 @@
 Intro
 
 Face research literature has outlined that the most relevant information is located in the upper part of the face, which triggers the largest brain electrical signal (e.g. N170). However several studies have shown that the human visual perception displays a low visual field (LVF) advantage. Zerouali et al. (2013) showed that fixating the top of a face elicits the highest N170, whether the face is presented upright or inverted. Suggesting that the upper-fixation bias is a consequence of LVF advantage. To follow-up, EEG and eye-tracking measurements were recorded, while participants viewed face-photographs, face-like phase-scrambled images and face-like checkerboards. ERPs were averaged by gaze location. We observed an optimal fixation position located on the upper-visual field, with N170 decreasing amplitude from the upper to the lower fixations. Such results suggest that N170 amplitude modulations arise from a retinotopic visual setting enhancing visual information located in the LVF of facial and non-facial stimuli.}
+
+
+Now it's my time to work on the intro and I'm adding this information.
```

Comment:
I think that this particular data is not relevant here, it should be on the method section